### PR TITLE
Add load tests for main CSV data module functions

### PR DESCRIPTION
**Pull request by SasinduDilshara**
Add load tests for main CSV data module functions
Fixes https://github.com/ballerina-platform/ballerina-library/issues/6756

### CODE REVIEW

**Comment by SasinduDilshara** [comment]
This PR needs to merge after https://github.com/ballerina-platform/module-ballerina-data.csv/pull/4

File: load-tests/parse-stream/src/app.bal [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (LakshanWeerasinghe) [inline load-tests/parse-stream/src/app.bal:1]: Need to update the licence headers.

File: load-tests/parse-stream/src/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+[build-options]
+observabilityIncluded = false
+cloud = "k8s"
+
+[[dependency]]
+org = "ballerina"
+name = "data.csv"
+version = "0.1.0"
+repository="local"
```
Comment (LakshanWeerasinghe) [inline load-tests/parse-stream/src/Ballerina.toml:14]:
```suggestion
repository="local"

```

File: load-tests/parse-stream/src/Ballerina.toml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[package]
+org = "wso2"
+name = "csv_parse_stream"
+version = "0.0.1"
+
+[build-options]
+observabilityIncluded = false
```
Comment (LakshanWeerasinghe) [inline load-tests/parse-stream/src/Ballerina.toml:7]:
```suggestion
```

File: load-tests/parse-stream/src/Ballerina.toml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+cloud = "k8s"
+
+[[dependency]]
```
Comment (LakshanWeerasinghe) [inline load-tests/parse-stream/src/Ballerina.toml:10]: We should remove this dependency from here. When running the load tests, this should always use the latest `data.csv` version.

File: load-tests/parse-string/src/app.bal [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/http;
+import ballerina/data.csv as csv;
```
Comment (LakshanWeerasinghe) [inline load-tests/parse-string/src/app.bal:19]:
```suggestion
import ballerina/data.csv;
```

Comment (LakshanWeerasinghe) [reply load-tests/parse-string/src/app.bal:19]: reorder the imports

File: load-tests/parse-string/src/app.bal [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+final string data = check io:fileReadString("resources/input.csv");
+
+isolated service http:Service /BalPerformance on new http:Listener(9090) {
```
Comment (LakshanWeerasinghe) [inline load-tests/parse-string/src/app.bal:23]:
```suggestion
isolated service /BalPerformance on new http:Listener(9090) {
```

**Comment by SasinduDilshara** [comment]
This PR will be merge after https://github.com/ballerina-platform/module-ballerina-data.csv/pull/4

**Comment by SasinduDilshara** [comment]
This PR will draft until https://github.com/ballerina-platform/module-ballerina-data.csv/pull/4 get merge

**Comment by SasinduDilshara** [comment]
This PR will be reopen after https://github.com/ballerina-platform/module-ballerina-data.csv/pull/4

**Pull request closed without merging**